docs/finetune_parameters.md

```diff
@@ -10,6 +10,7 @@ The following are the parameters supported in the finetuning workflow.
 |tokenizer_name|None|Path to pretrained tokenizer from huggingface.co/models. If not provided, the tokenizer will be loaded from the `base_model`.|
 |gpt_base_model|True|This parameter is for [Transformers#22482](https://github.com/huggingface/transformers/issues/22482). It needs to be set to True when the pretrained model is realted to gpt, otherwise it is False.|
 |output_dir|/tmp/llm-ray/output|The output directory to store the finetuned model.|
+|report_to|none|The list of integrations to report the results and logs to.|
```

Comment:
can you please list the supported options? Can you also show the picture for one of your runs?

Reply:
In my opinion, Do we only need to support one?  Because each tool needs to be pip. 

![image](https://github.com/intel/llm-on-ray/assets/45281494/df15810a-32cb-4161-9ae4-cbd27b10aa11)